mobile/src/main/java/com/truckmuncher/truckmuncher/vendor/VendorTrucksService.java

```diff
@@ -66,6 +66,7 @@ protected void onHandleIntent(Intent intent) {
                 values.put(PublicContract.Truck.KEYWORDS, Contract.convertListToString(truck.keywords));
                 values.put(PublicContract.Truck.COLOR_PRIMARY, truck.primaryColor);
                 values.put(PublicContract.Truck.COLOR_SECONDARY, truck.secondaryColor);
+                values.put(PublicContract.Truck.OWNED_BY_CURRENT_USER, 1);
```

Comment:
put `true` instead of 1. Easier to read and the conversion will happen automatically.